src/scroll_accelerator/cli.py

```diff
@@ -0,0 +1,501 @@
+from __future__ import annotations
+
+import argparse
+import logging
+import os
+import sys
+from typing import Optional, Sequence
+
+import yaml
+
+from scroll_accelerator.constants import (
+    APP_DESCRIPTION,
+    APP_NAME,
+    CONFIG_DIR,
+    CONFIG_FILE,
+    DEFAULT_CONFIG,
+    DEFAULT_MACOS_CONFIG,
+    SYSTEMD_UNIT_NAME,
+)
+from scroll_accelerator.daemon import (
+    daemon_log,
+    daemon_status,
+    install_daemon,
+    restart_daemon,
+    stop_daemon,
+    uninstall_daemon,
+)
+from scroll_accelerator.scroll_accelerator import (
+    ScrollAccelerator,
+    check_config,
+)
+
+# Logging
+
+logger = logging.getLogger(__name__)
+
+
+def init_logging(verbose: int = 0):
+    """Initialize logging.
+
+    Args:
+        verbose: Verbosity level.
+    """
+    if verbose <= 0:
+        format = "%(levelname)s: %(message)s"
+    else:
+        format = "%(asctime)s %(levelname)s: %(message)s"
+
+    logging.basicConfig(
+        format=format,
+        level=max(1, logging.WARNING - verbose * 10),
+    )
+
+
+# Config
+
+
+def get_default_config() -> dict:
+    """Default config for the application.
+
+    This is probably very subjective. But just to have some reasonable defaults,
+    which might be used.
+
+    Returns:
+        Default config for the application.
+    """
+    if sys.platform == "darwin":
+        # Darwin (MacOSX) already has builtin mouse scroll wheel acceleration,
+        # so we should apply much less extra acceleration.
+        config = DEFAULT_MACOS_CONFIG.copy()
+    else:
+        # All other common desktop platforms do not (Linux, Windows).
+        # (Android does, but we don't expect that here...)
+        config = DEFAULT_CONFIG.copy()
+
+    try:
+        check_config(**config)
+    except Exception as e:
+        assert False, f"Default config is invalid: {e}."
+
+    return config
+
+
+def get_command_line_config(args: argparse.Namespace) -> dict:
+    """Get config from command line arguments.
+
+    Args:
+        args: Command line arguments.
+
+    Returns:
+        Config from command line arguments.
+    """
+    default_config = get_default_config()
+    config = {}
+    for key in default_config.keys():
```

Comment:
I don't understand why you need the default config here. Use sth like `CONFIG_KEYS` instead.
```suggestion
    config = {}
    for key in CONFIG_KEYS:
```